params/version.go

```diff
@@ -22,8 +22,8 @@ import (
 
 const (
 	VersionMajor = 1          // Major version component of the current release
-	VersionMinor = 9          // Minor version component of the current release
-	VersionPatch = 11         // Patch version component of the current release
+	VersionMinor = 10         // Minor version component of the current release
+	VersionPatch = 1          // Patch version component of the current release
```

Comment:
This diff looks funny :D You have to see individual commits to see the reason :D